src/libraries/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509ChainStatus.cs

```diff
@@ -4,13 +4,15 @@
 
 using System;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 
 namespace System.Security.Cryptography.X509Certificates
 {
     public struct X509ChainStatus
     {
         public X509ChainStatusFlags Status { get; set; }
 
+        [AllowNull]
```

Comment:
Isn't AllowNull normally for when the setter is going to turn null into something else?  I'd just say this is a normal not-null string; most people don't even know there's a setter.

Reply:
(I guess the getter is turning the null into something else; so this is fine... but I don't know that I'd bother making the distinction)